tests/package.json

```diff
@@ -1,6 +1,12 @@
 {
   "nativescript": {
-    "id": "org.nativescript.ngtests"
+    "id": "org.nativescript.ngtests",
+    "tns-ios": {
```

Comment:
Can you remove the platforms here?

Reply:
no, because than tns build will fail with `version of undefined` errors

Reply:
Ok, change the versions to 'next' then :)

Reply:
done ;)